Services/DataBaseService/Database/Migrations/Scripts/04.2020/0004_CreateUsersAvatarsTable.sql

```diff
@@ -1,7 +1,7 @@
 ﻿CREATE TABLE dbo.UsersAvatars (
 	[Id] UNIQUEIDENTIFIER UNIQUE NOT NULL,
-	[Avatar] VARBINARY(MAX) NOT NULL,
-	[AvatarExtension] NVARCHAR(20) NOT NULL,
```

Comment:
Is default avatar is null? I thought, we had a default picture

Reply:
Default avatar isn't nul. But it isn't necessary to save it for each user that hasn't avatar in the database. So I have default image in the wwwroot

Reply:
Okey, now I understand this